YakimS/assignment1/Q1

```diff
@@ -0,0 +1,71 @@
+public class Assignment1Q1
```

Comment:
It would be better to give this class a name that would indicate what it does.

Reply:
Sure. Is "AnagramValidityChecker" would do?